server/settings.py

```diff
@@ -185,6 +185,9 @@ def env(variable, fallback_value=None):
 S3_USE_HTTPS = False
 FLASK_ASSETS_USE_S3 = False
 USE_S3 = FLASK_ASSETS_USE_S3
+RECAPTCHA_PUBLIC_KEY = env('RECAPTCHA_PUBLIC_KEY', '')
```

Comment:
I don't think we need these empty string as default value.
It's better to keep `None`